pom.xml

```diff
@@ -151,9 +153,36 @@
       <version>${wiremock.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>net.sourceforge.htmlunit</groupId>
+      <artifactId>htmlunit</artifactId>
+      <version>${htmlunit.version}</version>
+    </dependency>
   </dependencies>
   <build>
     <plugins>
+      <!-- Maven Exec Plugin to run frontend build -->
+      <plugin>
```

Comment:
I think it's better to have the possibility to disable it with a flag in a profile. So that it doesn't run every time during development.